### PR TITLE
added Log

### PR DESCRIPTION
**Pull request by AnnaDoroshenko**
Check this out

### CODE REVIEW

File: src/Log.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#include "Log.h"
+
+
+Log::Log() : Log(OutputLevel::None) {}
+
+
+Log::Log(OutputLevel outputLevel) : m_ClassName(""), m_FuncName("") {
```
Comment (IgorBoyarshin) [inline src/Log.cpp]:
```suggestion
Log::Log(OutputLevel outputLevel) : m_ClassName(""), m_FuncName(""), m_LogLevel(LogLevel::Info) {
```

Comment (IgorBoyarshin) [reply src/Log.cpp]: Also could move the body of the constructor here as well

File: src/main.cpp [hunk 2]
```diff
@@ -2,11 +2,16 @@
...
 #include "Client.h"
 #include "Server.h"
+#include "Log.h"
 
 
 int main() {
     std::cout << "---------- Start ----------" << std::endl;
 
+    Log log;
+    std::cout << log.setLevel(Log::LogLevel::Info) << "test" << std::endl;
```
Comment (IgorBoyarshin) [inline src/main.cpp]: Don't want to deal with std::cout. Make Log self-sufficient: 
`log.setLevel(...) << ...`

**Review by IgorBoyarshin — CHANGES_REQUESTED** [review]
Fix the specified things

**Comment by AnnaDoroshenko** [comment]
Done. Check out new version.

**Review by IgorBoyarshin — APPROVED** [review]

**Pull request merged**